src/App.tsx

```diff
@@ -11,11 +11,23 @@ export class App extends React.Component<{}, State> {
   };
 
   addOne = () => {
-    // write code here
+    this.setState(state => {
+      const newCounter = state.count;
+
+      return {
+        count: newCounter + 1,
+      };
+    });
   };
 
   add100 = () => {
-    // write code here
+    this.setState(state => {
+      const newCounter = state.count;
+
+      return {
+        count: newCounter + 100,
+      };
```

Comment:
```suggestion
      return {
        count: state.count + 100,
      };
```
you can simplify your code

Reply:
Thanks